Assets/Scripts/PlayerController.cs

```diff
@@ -41,16 +68,16 @@ private void Update()
             Move();
         }
 
-        if (_onGround && Input.GetAxis("Jump") == 1.0f)
+        if (Input.GetAxis("Jump") == 1.0f)
         {
-            Jump();
+            if (IsOnGround())
+                Jump();
         }
```

Comment:
Any reason in particular for separating the this from the if statement above? I think it would be better to have it in the one line to reduce nesting.

Reply:
can also. i was thinking it's better to separate, because the input is a player action, while isonground is not, so to me they feel like putting them together feels a bit weird